tutorials/weaviate_llama_index_search_and_retrieval.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Is there a class method that enables you to initialize <code>WeaviateVectorStore</code> without supplying the client directly? Maybe a <code>from_defaults</code> class method?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Not really as I am assigning to a llama_index WeaviateVectorStore object the weaviate client that was created above.</p>